lib/xandra/protocol.ex

```diff
@@ -42,7 +42,7 @@ defmodule Xandra.Protocol do
     body =
       <<byte_size(id)::16>> <>
       id <>
-      encode_params(columns, values, options, true)
+      encode_params(columns, values, options, prepared.result_columns != nil)
```

Comment:
`_skip_metadata? = not is_nil(prepared.result_columns)`? The `not is_nil` is a personal preference, but `_skip_metadata?` is quite helpful IMO, passing booleans to functions is always a "mysterious" thing to do IMO. Wdyt?